Add MeritSun LFP 200 as supported (#30)
syssi/esphome-pace-bms: README.md

* Revov R100
* Revov R9
* Greenrich U-P5000

> * MeritSun LFP 200 - 48V (After activating Modbus with pbmstool.exe)

Reviewer: ```suggestion
* MeritSun LFP 200 - 48V (Modbus must be enabled by pbmstool.exe)
```
